frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  List,
+  Box,
+  AppBar,
+  Toolbar,
+  IconButton,
+  Button,
+  SwipeableDrawer,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import HomeIcon from '@material-ui/icons/Home';
+import AccountTreeIcon from '@material-ui/icons/AccountTree';
+import CreateIcon from '@material-ui/icons/Create';
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import useStyles from './SidebarStyle';
+
+type Anchor = 'left';
+
+const Sidebar = (props: { history: any }) => {
+  const { history } = props;
+
+  const [state, setState] = React.useState({
+    left: false,
+  });
+
+  const toggleDrawer = (anchor: Anchor, open: boolean) => (event: React.KeyboardEvent | React.MouseEvent) => {
+    if (
+      event &&
+      event.type === 'keydown' &&
+      ((event as React.KeyboardEvent).key === 'Tab' || (event as React.KeyboardEvent).key === 'Shift')
+    ) {
+      return;
+    }
+
+    setState({ ...state, [anchor]: open });
+  };
+
+  const classes = useStyles();
+
+  const itemsList = [
+    {
+      text: 'Homepage',
+      icon: <HomeIcon className={classes.icon} />,
+      onClick: () => history.push('/'),
+    },
+    {
+      text: 'Projects',
+      icon: <AccountTreeIcon className={classes.icon} />,
+      onClick: () => history.push('/projects'),
+    },
+    {
+      text: 'Create Business Account',
+      icon: <CreateIcon className={classes.icon} />,
+      onClick: () => history.push('/businessAccount/new'),
+    },
+    {
+      text: 'Create Client Account',
+      icon: <CreateIcon className={classes.icon} />,
+      onClick: () => history.push('/clientAccount/new'),
+    },
+    {
+      text: 'Create Employee Account',
+      icon: <CreateIcon className={classes.icon} />,
+      onClick: () => history.push('/employeeAccount/new'),
+    },
+  ];
+
+  function handleClick() {
```

Comment:
use arrow function instead of functions.

Reply:
Done!